webui/react/src/pages/ExperimentDetails/TrialsComparisonModal.tsx

```diff
@@ -76,9 +88,10 @@ export const TrialsComparisonTable: React.FC<TableProps> = ({
   trialIds,
   trials,
   experiment,
+  runs,
   onUnselect,
 }: TableProps) => {
-  const [trialsDetails, setTrialsDetails] = useState(trials ?? []);
+  const [trialsDetails, setTrialsDetails] = useState<(TrialDetails | FlatRun)[]>(trials ?? []);
```

Comment:
shouldn't this be `TrialDetails[] | FlatRun[]` (i.e. an array of `TrialDetails` or an array of `FlatRun`s)?

Reply:
If I recall correctly I did it this way because of a type error, but let me go in a see if I can fix it.